src/data_operations.py

```
@@ -35,6 +43,36 @@ def process_sofr_data(sofr_series, fred, col_date, dataPath, SAVE_AS_PICKLE, PUS
         upload_combined_to_gcp(sofr_data, "sofr/sofr_data", bucket, SAVE_AS_PICKLE)
     return audit_data
 
+def process_m1m2_data(fred, col_date, dataPath, SAVE_AS_PICKLE, PUSH_TO_GCP, bucket):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_m1m2_data`

(ANN201)